pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -15,7 +15,6 @@
 , sendEmailSupport
 , darwin
 , nixosTests
-, withLibsecret ? false
 , pkg-config, glib, libsecret
```

Comment:
Please add a comment that `libsecret` is nullable so that someone does not accidentally remove the `optionals` in the future.

Reply:
I put `libsecret?null` in there (that's what's also common for other packages), and placed a comment into the commit. If I should add something more, please tell me what would be appropriate.

Reply:
@SuperSandro2000 and other people tend to clean up the `? null` since it is actually useless in Nixpkgs – the attribute will always be present even if you override it with `null` in an overlay. So a comment in the vein of `xxx can be passed null to disable feature yyy` might be cleaner.

Reply:
> that's what's also common for other packages

and quite ugly. It is nicer to have a boolean flag.